psydac/api/ast/parser.py

```diff
@@ -5,22 +5,24 @@
 from sympy import Add
 from sympy import Abs
 from sympy import Symbol, Idx
+from sympy import Max
+from sympy import Basic
+from sympy.simplify import cse_main
```

Comment:
It appears that this function is not currently used in the module. Do you still want to import it?

Reply:
ah I forgot to push my commits